browser/blueprints.py

```diff
@@ -147,96 +109,44 @@ def change_display(project_id, display_attribute, value):
     """
     start = timeit.default_timer()
 
-    try:
-        project = Project.get(project_id)
-        if not project:
-            return jsonify({'error': 'project_id not found'}), 404
-        change = ChangeDisplay(project)
-        payload = change.change(display_attribute, value)
-        project.update()
-
-    except Exception as e:  # TODO: more error handling to identify problem
-        traceback.print_exc()
-        return jsonify({'error': str(e)}), 500
+    project = Project.get(token)
+    if not project:
+        return abort(404, description=f'project {token} not found')
+    change = ChangeDisplay(project)
+    payload = change.change(display_attribute, value)
+    project.update()
 
     current_app.logger.debug('Changed to %s %s for project %s in %s s.',
-                             display_attribute, value, project_id,
+                             display_attribute, value, token,
                              timeit.default_timer() - start)
     return jsonify(payload)
 
 
-@bp.route('/undo/<int:project_id>', methods=['POST'])
-def undo(project_id):
+@bp.route('/undo/<token>', methods=['POST'])
+def undo(token):
     start = timeit.default_timer()
-    try:
-        project = Project.get(project_id)
-        if not project:
-            return jsonify({'error': 'project_id not found'}), 404
-        payload = project.undo()
-    except Exception as e:  # TODO: more error handling to identify problem
-        traceback.print_exc()
-        return jsonify({'error': str(e)}), 500
-
-    current_app.logger.debug('Undid action for project %s finished in %s s.',
-                             project_id, timeit.default_timer() - start)
-    return jsonify(payload)
 
+    project = Project.get(token)
+    if not project:
+        return abort(404, description=f'project {token} not found')
+    payload = project.undo()
 
-@bp.route('/redo/<int:project_id>', methods=['POST'])
-def redo(project_id):
-    start = timeit.default_timer()
-    try:
-        project = Project.get(project_id)
-        if not project:
-            return jsonify({'error': 'project_id not found'}), 404
-        payload = project.redo()
-    except Exception as e:  # TODO: more error handling to identify problem
-        traceback.print_exc()
-        return jsonify({'error': str(e)}), 500
-
-    current_app.logger.debug('Redid action for project %s finished in %s s.',
-                             project_id, timeit.default_timer() - start)
+    current_app.logger.debug('Undid action for project %s finished in %s s.',
+                             token, timeit.default_timer() - start)
     return jsonify(payload)
 
 
-@bp.route('/load/<bucket>/<filename>', methods=['POST'])
-def load(bucket, filename):
-    """
-    Initate TrackEdit/ZStackEdit object and load object to database.
-    Send specific attributes of the object to the .js file.
-    """
+@bp.route('/redo/<token>', methods=['POST'])
```

Comment:
What do you think about prefixing all the JSON API routes with `/api`?

This would more clearly differentiate between JSON routes and template routes.

Reply:
I like that change, will do now